### ISSUE TITLE
Enable resolution choice before lauching video

### ISSUE THREAD

**Issue by Silmathoron** [Chocobozzz/PeerTube#3704]
**Describe the problem to be solved**
Currently, one has to launch the video before being able to choose a resolution.
Due to the federated nature of PeerTube, this means that, potentially, high resolution videos will start being streamed before the person can switch to a lower resolution (since one cannot expect users to always think of changing default resolution in each and every instance they use).
This leads to:

* undesired data use on the user's side
* useless load on the server side
* general waste of energy

**Describe the solution you would like:**

Make the video "menu bar" visible before launching the video in addition to the play button so that we can do everything before launching the video (select resolution, mute...).

Basically something that would look like that *from the start* (before the video is launched):

![image](https://user-images.githubusercontent.com/8523051/107054710-c58ad280-67d0-11eb-88bc-52e984c0c4dd.png)

**Comment by Alterak** [comment]
When the video starts playing, the lowest resolution is selected, then the resolution is automatically increased as much as your bandwidth allows.
I have good internet, but when I start watching a video, it is not immediately reproduced in high definition, but only after a while.

**Comment by Silmathoron** [comment]
Thanks for the answer, unfortunately that does not really solve the problem:

1. a user may have 4G/LTE with a limited data plan
2. personaly, I want to be able to limit the impact of the videos I watch by setting them at an intermediate or low quality despite my large bandwidth

**Comment by Alterak** [comment]
> and some people may want to be able to mute a video before starting

You can set this feature in your account settings
![Screenshot_1](https://user-images.githubusercontent.com/11453567/107063835-e3b1fc00-67eb-11eb-8ef2-857c66ba5c4b.png)

And you can change the quality of the video after the start of playback.
In any case, I see no inconvenience in this.

**Comment by bkil** [comment]
I would also like this. Also related: #1909, #3874, #3233
